charts/query-exporter/templates/configmap-cert-db.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "query-exporter.fullname" . }}-cert-db
+  labels:
+    {{ include "query-exporter.labels" . | nindent 4 }}
+data:
+ query-exporter-cert-db.pem: |-
+  {{ tpl .Values.cert_db . | nindent 4 }}
```

Comment:
I added a new file to create a config map
1. Checking config maps
```
kubectl get configmaps -n p1-metrics | grep "query-exporter"
query-exporter           1      88m
query-exporter-cert-db   1      3m
```
2. Describing config map
kubectl describe configmap/query-exporter-cert-db -n p1-metrics
Name:         query-exporter-cert-db
Namespace:    p1-metrics
Labels:       app.kubernetes.io/instance=query-exporter
              app.kubernetes.io/managed-by=Helm
              app.kubernetes.io/name=query-exporter
              app.kubernetes.io/version=2.7.0
              helm.sh/chart=query-exporter-0.1.1
Annotations:  meta.helm.sh/release-name: query-exporter
              meta.helm.sh/release-namespace: p1-metrics

Data
====
query-exporter-cert-db.pem:
----

-----BEGIN CERTIFICATE-----
MIIEBzCCAu+gAwIBAgICEAAwDQYJKoZIhvcNAQELBQAwgZQxCzAJBgNVBAYTAlVT
MRAwDgYDVQQHDAdTZWF0dGxlMRMwEQYDVQQIDApXYXNoaW5ndG9uMSIwIAYDVQQK
DBlBbWF6b24gV2ViIFNlcnZpY2VzLCBJbmMuMRMwEQYDVQQLDApBbWF6b24gUkRT
MSUwIwYDVQQDDBxBbWF6b24gUkRTIGFwLWVhc3QtMSBSb290IENBMB4XDTE5MDIx
NzAyNDcwMFoXDTIyMDYwMTEyMDAwMFowgY8xCzAJBgNVBAYTAlVTMRMwEQYDVQQI
deleted for posted proposals
-----END CERTIFICATE-----

BinaryData
====

Events:  <none>